articles/root-cause-analysis.md

```diff
@@ -0,0 +1,79 @@
+---
+authorName: Matthias Meid
+authorGithubUsername: matthias-meid
+issue: 98
+title: You Always Have Time for a Proper Root Cause Analysis
+---
+# {{page.title}}
+
+A production issue will pop up, even in solid software. [Don't panic.](https://en.wikipedia.org/wiki/Phrases_from_The_Hitchhiker%27s_Guide_to_the_Galaxy#Don't_Panic)
+
+We're more than eager to resolve the issue quickly. We should be, yet we should never stop being careful and analytical. Otherwise we might deploy another embarassing bug or suppress the problem without solving it, making everything worse.
+
+Saving an hour of analysis is not worth it.
+
+## People Will Describe the Symptoms
+
+Keep in mind that users will report _symptoms_. This has nothing to do with a lack of understanding or skill. They have an outside view.
+
+What you need to find, and then fix, is the _problem_ (or more precisely its _cause_).
+
+Don't jump to conclusions. Listen to all the clues. Analyse. Re-produce. Re-think. Refine (or create) the ticket. Think of test cases which are explicit about expected and actual results.
```

Comment:
Think of test cases which are explicit about expected and actual results. -> and implement them.